### PR TITLE
Rate limiting

### PR DESCRIPTION
**Pull request by michellefernando9**

### CODE REVIEW

File: tyk-docs/content/tyk-cloud/tyk-cloud.md [hunk 1]
```diff
@@ -1,74 +0,0 @@
----
-title: "Tyk Cloud"
```
Comment (pagemag) [inline tyk-docs/content/tyk-cloud/tyk-cloud.md]: @michellefernando9 , Is this article meant to be deleted?

**Review by pagemag — COMMENTED** [review]
The Throttling and rate limiting changes are okay. Just got a question if tyk-cloud article is meant to be deleted?

File: tyk-docs/content/tyk-cloud/tyk-cloud.md (hunk 1 above)
Comment (michellefernando9) [reply tyk-docs/content/tyk-cloud/tyk-cloud.md]: @pagemag This has been added to the docs again.

**Review by pagemag — APPROVED** [review]
Approved!

**Review by marksou — APPROVED** [review]

**Comment by letzya** [comment]
@michellefernando9 we have 2 similar  PRs https://github.com/TykTechnologies/tyk-docs/pull/2058, can you make them into one please?

**Comment by letzya** [comment]
@michellefernando9 please check what we need from this PR as it's similar to https://github.com/TykTechnologies/tyk-docs/pull/2058 that was merged

**Review by letzya — CHANGES_REQUESTED** [review]
@michellefernando9 we have 2 similar PRs https://github.com/TykTechnologies/tyk-docs/pull/2058, can you make them into one please?

**Review by letzya — CHANGES_REQUESTED** [review]
per discussion

**Comment by michellefernando9** [comment]
Now that the changes have been made, I don't see the point in creating a new PR. Would be like doing the same work over and over again.

**Review by marksou — APPROVED** [review]

**Comment by letzya** [comment]
fixed in another pr

**Pull request closed without merging**